### DISCUSSION TITLE
install on laravel 7.2 with error

### DISCUSSION THREAD

**Discussion by amirhossein-ceh** [spatie/laravel-sitemap#403] [Q&A]
i wanna install it on my project with laravel 7.2
so i got this error : 

 Problem 1
    - spatie/laravel-sitemap[5.9.0, ..., 5.9.2] require illuminate/support ^8.0 -> found illuminate/support[v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires spatie/laravel-sitemap ^5.9 -> satisfiable by spatie/laravel-sitemap[5.9.0, 5.9.1, 5.9.2].

Could you please help me to solve this
thanks.

**Comment by ousid** [comment]
Hey, @amirhossein-ceh 

Please take a look at this thread:
https://github.com/spatie/laravel-sitemap/discussions/400
